src/main/java/qna/domain/Answer.java

```diff
@@ -60,6 +63,15 @@ protected Answer() {
 
     }
 
+    public DeleteHistory delete(User loginUser) throws CannotDeleteException {
```

Comment:
```suggestion
    public DeleteHistory delete(User loginUser) {
```
외부에서 강제로 예외처리하도록 만들 필요는 없어보이네요

Reply:
네 Checked Exception -> UnChecked Exception으로 변경했을 경우에는 해당 예외클래스를 검색해서 모두 제거해주는것이 좋을것 같습니다.